src/Model/Scope.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\data\Model;
+
+use atk4\data\Exception;
+use atk4\data\Model;
+
+/**
+ * The root scope object used in the Model::$scope property
+ * All other conditions of the Model object are elements of the root scope
+ * Scope elements are joined only using AND junction.
+ */
+class Scope extends Scope\CompoundCondition
+{
+    /**
+     * @var Model
+     */
+    protected $model;
+
+    public function __construct()
+    {
+        parent::__construct(...func_get_args());
+
+        $this->junction = self::AND;
```

Comment:
we should definitelly define method param excl. the possibility to set junction

Reply:
What do you mean exactly?

Reply:
declare function with `(array $nestedConditions = []`

and call ˙parent::__constructor($nestedConditions, self::AND)˙, better, right?

Reply:
It is better. Done now.